src/node_report.cc

```diff
@@ -793,11 +793,24 @@ static void PrintComponentVersions(JSONWriter* writer) {
   std::stringstream buf;
 
   writer->json_objectstart("componentVersions");
+  std::vector<std::pair<std::string, std::string>> versions_vec;
```

Comment:
Performance isn't as important here but consider using the same technique for consistency.

Reply:
I think `json_keyvalue(k, v)` has no implement for std::string_view